cli/config/parser.go

```diff
@@ -128,6 +108,32 @@ func createCluster(clusterName string, parsedKeyVal map[string]string) (*PxfClus
 	return &cluster, nil
 }
 
+// createClusterMap creates a map of properties and corresponding values given the file contents of the cluster file.
+func createClusterMap(fileContents []string) (map[string]string, error) {
+	clusterMap := make(map[string]string)
+	for lineNum, line := range fileContents {
+		line = strings.TrimSpace(strings.ToLower(line))
+		// Remove any line that is a TOML comment
+		if line == "" || line[0] == '#' {
+			continue
+		}
+		lineKeyVal := strings.Split(line, "=")
+		// lineKeyVal[0] = key, lineKeyVal[1] = value, anything else is invalid
+		if len(lineKeyVal) != 2 {
+			return nil, fmt.Errorf("found more than one character '=' on line %d: %s", lineNum+1, line)
```

Comment:
```suggestion
			return nil, fmt.Errorf("found more than one character '=' on line %d: %s", lineNum + 1, line)
```

nit: consistency of spaces between operators here and lines below

Reply:
`lineNum+1` is enforced by `gofmt` because they want to show operator precedence (sorta) and enforce consistency.

Here is the Go source code that handles formatting: https://github.com/golang/go/blob/master/src/go/printer/nodes.go#L734

Email group discussion:
https://groups.google.com/g/golang-nuts/c/Uqu6IVLDSeI/m/7xpPLrPxMHAJ